.github/workflows/check-codegen.yml

```diff
@@ -14,6 +14,11 @@ jobs:
     - uses: actions/setup-go@v5
       with:
         go-version-file: 'go.mod'
+    - name: Download k8s.io/code-generator
```

Comment:
We used to install the `code-generator` via the `Makefile`:
```
.PHONY: deepcopy-gen
deepcopy-gen: $(DEEPCOPY_GEN) ## Download deepcopy-gen locally if necessary.
$(DEEPCOPY_GEN): $(LOCALBIN)
	test -s $(LOCALBIN)/deepcopy-gen || GOBIN=$(LOCALBIN) go install k8s.io/code-generator/cmd/deepcopy-gen@$(CODE_GENERATOR_VERSION)
```
I would suggest to keep it that way. If we don't need the individual generators like deepcopy etc. we can have a `code-generator` target in the `Makefile`.

Reply:
I think this additional step is no longer needed, so I've removed it from the workflow.